docs/.eslintrc.yml

```diff
@@ -0,0 +1,22 @@
+parser: babel-eslint
```

Comment:
cant we just have parent folder's eslintrc?

Reply:
Sure, I was just trying to have two different projects so that they are not depending. We can just move ```docs``` somewhere else and it would still work. Also, we can have different ```eslint``` rules for docs and the library.

Reply:
Should I remove ```prettier``` from ```docs``` too then?

Reply:
@prateekbh Just this last thing to clarify and we can merge.